docs/source/tutorial_usb_analysis.rst

```diff
@@ -0,0 +1,131 @@
+
+This tutorial walks through the whole process of running a USB protocol capture of a target device, in this case a keyboard. Hopefully most people have a USB keyboard available and can follow along,
+though this process is also applicable to any USB peripheral device.
+
+Prerequisites
+-------------
+
+ * Install the Cynthion tools by following :doc:`getting_started`
+ * Install Packetry by following `Getting Started with Packetry <https://packetry.readthedocs.io/en/latest/quick_start.html>`_
+ * Run the analyzer gateware on Cynthion by following :doc:`getting_started_packetry`
+
+Determine device speed
+----------------------
+
+USB 2.0 supports three different speeds: Low (1.5 Mbit/s), Full (12 Mbit/s), and High (480 Mbit/s).
+The analyzer needs to know what speed to expect, so we need to detemine what speed the target device is using.
+
+    .. note::
+
+        Soon this step won't be necessary, as the analyzer will be able to determine the speed automatically,
+        but that feature is currently in development.
+
+To determine the speed, we plug the target device into a host check what speed it reports. The way to check depends on the host operating system:
+
+.. tab:: Linux
+
+    Run the command ``sudo dmesg -W`` in a terminal window and then plug the target device in, some new lines should show up with information about it::
+
+        [975321.743878] usb 1-6.1: new full-speed USB device number 12 using xhci_hcd
+        [975321.821329] usb 1-6.1: New USB device found, idVendor=3434, idProduct=0108, bcdDevice= 2.00
+        [975321.821341] usb 1-6.1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
+        [975321.821345] usb 1-6.1: Product: Keychron Q1
+        [975321.821348] usb 1-6.1: Manufacturer: Keychron
+
+    Here, it shows that the target device has enumerated at full-speed.
+
+
+.. tab:: macOS
+
+    Go to the Apple menu -> About This Mac -> Hardware -> USB. Highlight the target device and check the Speed field.
+
+.. tab:: Windows
+
+    The easiest way to look at USB device information on Windows is to install and run `USBView <https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/usbview>`_,
+    then find the target device and check its `Device Bus Speed` field.
+
+
+Connect
+-------
+
+Next, we'll connect everything up for capture. Within the Cynthion hardware the `Target-C` and `Target-A` ports are connected together,
+and the analysis gateware will capture any packets that are seen going between those ports. These packets are then sent out through the `Control` port.
+
+.. image:: ../images/cynthion-connections-packetry.svg
+  :alt: Connection diagram for using Cynthion with Packetry.
+
+First, connect the `Control` port to the host that will be running Packetry. Next, connect the `Target-C` port to a host.
+This can be a different host to the one running Packetry, or it can be the same host. If it is the same host, the two connections must not be on the same hub.
```

Comment:
Maybe this is a British/American thing, but I would say "different host than the one" instead of "different host to the one".

Reply:
I think it must be, because to me as a British speaker that looks fine but "different host than" sounds completely wrong.

What about "different host from"? That would read find to me but I don't know if it's valid in American English.

Reply:
"from" sounds good to me.

Reply:
Assuming it's OK, I went for "from"